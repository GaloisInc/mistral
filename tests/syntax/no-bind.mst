module Main where

f : Int
