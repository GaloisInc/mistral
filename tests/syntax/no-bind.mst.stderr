 
[error] tests/syntax/no-bind.mst:3,1-3,2
  signature, but no binding, for symbol `f`
